src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -94,7 +109,9 @@ class Lambda extends RequestStreamHandler {
       )
```

Comment:
Do you think it's worth adding a log for "Writing to the output stream"?

Reply:
Writing to the output stream is just a way of returning data from the lambda. I was trying to keep the log lines logging things related to the ingest rather than implementation details like this one. And if it fails, we'll get an exception in the logs and a step function failure because there will be no output.